src/classes/CanvasRenderingContext2D.js

```diff
@@ -129,7 +140,8 @@ export default class CanvasRenderingContext2D {
    * This array keeps track of the current path, so that fill and stroke operations can store the
    * path.
    */
-  _path = [createCanvasEvent('beginPath', [1, 0, 0, 1, 0, 0], {})];
+  // TODO why is the initial _path array not empty?
+  _path = [createCanvasEvent('beginPath', undefined, {})];
```

Comment:
@jtenner can you explain why the `_path` array is not empty?

at least in the Path2D class, there is no method `beginPath`
but also an empty context expects me to explicitly call `beginPath`, no?

Reply:
This was a design choice, and I agree. This probably isn't correct. However, I chose to keep things consistent between `CanvasRenderingContext2D` class and the `Path` class. I could be convinced to make a change on the `Path2D` class, but I'm not entirely sure it would be helpful.

Reply:
its helpful to get an \*exact\* record of API calls, which can be replayed to a live canvas

surely i can work around this issue in my consumer (ignore `path.beginPath()`), but .. 
this \*should\* be fixed upstream

> This was a design choice

why? why should we record calls that were not explicitly made?
again, i expect this tool to produce an \*exact\* image of my API calls